package.json

```diff
@@ -1,50 +1,56 @@
 {
-  "name": "sequelize-sscce",
-  "description": "SSCCE repository for Sequelize",
-  "version": "0.0.0-development",
+  "private": true,
   "engines": {
-    "node": ">=10.0.0"
+    "node": ">=12.0.0"
```

Comment:
```suggestion
    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```
In line with the main repo

Reply:
Doesn't it have to be 14 as pointed out in the description?

Reply:
I changed it back to node 10 until Sequelize 6 is EOL :)